backend/globaleaks/models/__init__.py

```diff
@@ -884,11 +946,25 @@ class _Subscriber(Model):
     registration_date = Column(DateTime, default=datetime_now, nullable=False)
     tos1 = Column(UnicodeText, default='', nullable=False)
     tos2 = Column(UnicodeText, default='', nullable=False)
+    creation_date = Column(DateTime, default=datetime_now, nullable=False)
+    state = Column(Integer, default=None, nullable=True)
+    organization_email = Column(UnicodeText, nullable=True)
+    organization_institutional_site = Column(UnicodeText, default='', nullable=False)
+    accreditation_date = Column(DateTime, nullable=True)
+    admin_name = Column(UnicodeText, nullable=True)
+    admin_surname = Column(UnicodeText, nullable=True)
+    admin_email = Column(UnicodeText, nullable=True)
+    admin_fiscal_code = Column(UnicodeText, nullable=True)
+    recipient_fiscal_code = Column(UnicodeText, nullable=True)
+    sharing_id = Column(UnicodeText(36), nullable=False, default=uuid4)
```

Comment:
TODO: remove, already subscriber.tid equals tenant.id acting as a unique id and reference making this attribute fully redundant.